### DISCUSSION TITLE
AIO interface wrong redirect

### DISCUSSION THREAD

**Discussion by naftali100** [nextcloud/all-in-one#893] [Questions] [answered]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. install normaly
2. press the "open AIO interface" button

### Expected behavior 
redirect to https://domain:8443
### Actual behavior 
redirect to https://ip-addr:8443

### Host OS 
ubuntu 20.04 arm

#### Nextcloud AIO version 
v1.4.2
#### Current channel 
latest-arm64
#### Other valuable info 
there is already a closed issue about this (#88)

**Comment by szaimen** [comment, marked as answer]
Hi, did you see this?
https://github.com/nextcloud/all-in-one#how-to-easily-log-in-to-the-aio-interface
![Screenshot_20220706-085302_Brave](https://user-images.githubusercontent.com/42591237/177487735-7127a30c-2467-4c47-bf42-db1e6915b8c0.png)

**Reply by naftali100** [reply]
oh... i missed that note. thank you
